test/ossl_store_test.c

```diff
@@ -76,9 +77,10 @@ static int get_params(const char *uri, const char *type)
     OSSL_STORE_close(ctx);
 
     if (pkey != NULL)
-        return EVP_PKEY_is_a(pkey, type);
+        ret = EVP_PKEY_is_a(pkey, type);
+    EVP_PKEY_free(pkey);
 
-    return 0;
+    return ret;
```

Comment:
The `err` section doesn't look very "errory" any more.  How about changing all from `err:` to this?

``` C
    if (pkey != NULL)
        ret = EVP_PKEY_is_a(pkey, type);
    EVP_PKEY_free(pkey);

 err:
    OSSL_STORE_close(ctx);
    return ret;
```

Reply:
Done